packages/client/src/components/app/GridBlock.svelte

```diff
@@ -50,8 +51,7 @@
       metadata: { dataSource: table },
     },
   ]
-  $: height = $component.styles?.normal?.height || "408px"
-  $: styles = getSanitisedStyles($component.styles)
+  $: gridContext?.minHeight?.subscribe($height => (minHeight = $height))
```

Comment:
The core grid is the source of truth and makes min height available in context, which we can consume from here and honour.

Reply:
Does unsubscribe ever need to be called on this?

Would storing minHeight as reactive variable and then assigning it to minHeight work?

```
$: contextMinHeight = gridContext.minHeight;
$: minHeight = $contextMinHeight
```

Obviously more verbose, but if the unsubscribe is necessary might be worth it.

Very much NAB anyway.

Reply:
Going to move this into the onMount handler to be more explicit about the fact it'll only run once. Makes sense to be explicit about it.